pandas/tseries/base.py

```diff
@@ -17,6 +17,26 @@
 import pandas.algos as _algos
 
 
+
+class DatelikeOps(object):
+    """ common ops for DatetimeIndex/PeriodIndex, but not TimedeltaIndex """
+
+    def strftime(self, date_format):
+        """
+        Return a list of formmatted strings specified by date_format
+
+        Parameters
+        ----------
+        date_format : str
+            date format string (e.g. "%Y-%m-%d")
+
+        Returns
+        -------
+        a list formatted strings
+        """
+        return self.format(date_format=date_format)
```

Comment:
yeh, this is because ``.format`` returns a list (to avoid some weird truncation str/numpy.str issues). I think you could do: ``return np.asarray(self.format(date_format))`` (and the ``.dt`` accessor wraps this anyhow, so the output would be the same. but @jorisvandenbossche is right, for consistency would be a bit better.